### DISCUSSION TITLE
Adding new translations

### DISCUSSION THREAD

**Discussion by bcssov** [bcssov/IronyModManager#231] [General]
Currently exploring gitlocalize and transifex to allow crowdfunding translation: https://explore.transifex.com/bcssov/irony-mod-manager/
Want to help out, check it out.

I don't have a habit to turn down a contribution. If you'd like to add a language manually and translate the old fashioned way here's what to do. You can always leave a note here or on discord for me to add another language on transifex:

Here's how to make a localization visible in Irony
1. Find your desired localization code here: https://lonewolfonline.net/list-net-culture-country-codes/
2. Locate your desired country
3. If you want to make a general (language neutral) translation pick the "Two Letter Lang Code" (ex. "es")
4. If you want to make a country specific translation such as for example Mexican you'll want to take "CultureInfo Code" column (ex. es-MX)
NOTE: Irony will acknowledge both
5. Navigate to Localization folder in Irony install
6. Take en.json and rename to your desired code (ex. es.json or es-MX.json)
7. Translate the values in the json
8. (OPTIONAL) If your translation does not render please contact me so I can add a font which supports rendering the alphabet (ex. japanese language)

IMPORTANT: When you translate do NOT translate anything inside curly braces inside strings. These are used for **formatting**.

**Comment by Magnus111111** [comment]
I'm trying to translate into Korean, but there is no board on transifex.

**Reply by bcssov** [reply]
Oh, this is a very old post indeed. I pretty much gave up on transifex. If you'd like I'll create a new translation subproject for korean on transifex for you and update translation files. All other contributors seems to prefer to directly edit json files. Let me know if you'd like me to bring transifex back to life.
